src/actions/events.js

```diff
@@ -0,0 +1,37 @@
+export const addEvent = ({
+  title = "",
+  description = "",
+  organisator = "",
+  city = "",
+  category = "",
+  image = false,
+  createdAt = 0,
+  startDate = 0,
+  endDate = 0,
+  id = "",
+} = {}) => ({
+  type: "ADD_EVENT",
+  event: {
+    title,
+    description,
+    organisator,
+    city,
+    category,
+    image,
+    createdAt,
+    startDate,
+    endDate,
+    id,
+  },
+});
+
+export const editEvent = (id, updates) => ({
+  type: "EDIT_EVENT",
+  id,
+  updates,
+});
+
+export const removeEvent = ({ id } = {}) => ({
```

Comment:
If you need only `id` here, then why are you passing an object with that? You can pass only `id` as primitive and that would be sufficient.